webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
- [ ] Kindly set the `mode` option to `development` here. Otherwise, it throws a warning that it will fall back to production mode. :pray:

Reply:
Hi @henatan99

Thanks for the review.

I have fixed the changes.

- Bootstrap was missing the @popper.js/core. However, I am not sure why it did not work on your local machine as it worked on deployment to Netlify.

- I have also set the mode to development for the webpack.config.js file